cmd/celestia/rpc.go

```diff
@@ -61,13 +64,34 @@ func init() {
 		false,
 		"Print JSON-RPC request along with the response",
 	)
+	blobCmd.AddCommand(getCmd, getAllCmd, submitCmd, getProofCmd)
+	rpcCmd.AddCommand(blobCmd)
```

Comment:
Why not make blob a separate cmd then? It's not an part of general purpose rpc command anymore, rather standalone cmd.

Reply:
Hmmm... not sure this is good UX if only one of the modules isn't an 'rpc subcommand'. I would +1 sticking to the current way slava has implemented it

Reply:
Agree. We have a 'rpc' command that that allows to interacts with different modules through the rpc.

Reply:
I think it's actually fine to separate blob out as it's going to be the most-used and would decrease surface area of the command that needs to be doc'd and kept up to date.

Reply:
* Consider the case where we fully migrate to cmds for each module. In such case `rpc` becomes redundant. * It makes sense to have `rpc` when the module namespace is generic.
* Blob is what the user wants, RPC is a way of accessing the API. Which can be HTTP or IPC. Would you put `http` there instead?

Reply:
I don't have objections to doing this once we migrate to cmdc from each module. But for now it does not look good: we have rpc command with all modules and we have a blob command that lives separately somewhere in the space.
Regarding 2: why don't we consider the case when a user would want to get data through the http and not ipc? then we should have 2 global commands http/ipc.